evalml/automl/automl_search.py

```diff
@@ -764,6 +757,14 @@ def _post_evaluation_callback(self, pipeline, evaluation_results):
             self.add_result_callback(self._results['pipeline_results'][pipeline_id], pipeline, self)
         return pipeline_id
 
+    def _check_for_high_variance(self, pipeline, cv_scores, threshold=0.2):
```

Comment:
nitpick, but should we enforce that users can't pass in `threshold < 0`, even though this is a private method?